src/jit/lsra.h

```diff
@@ -1354,6 +1381,23 @@ class RefPosition
 
     unsigned        getMultiRegIdx() { return multiRegIdx;  }
 
+    // Returns true if codegen has indicated that the tree node
+    // referred to by RefPosition can be treated as a contained
+    // memory operand if no register was allocated.
+    bool           RequiresRegOptionally()
```

Comment:
I would prefer a less verbose name here - I think it would be logical for it to be the same as the method on the node - i.e. `IsRegOptional`